ramhorns-derive/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub fn content_derive(input: TokenStream) -> TokenStream {
             let mut rename = None;
             let mut skip = false;
 
-            let mut parse_attr = |attr: &Attribute| -> Result<(), Error> {
+            let mut parse_attr = |attr: &Attribute| -> std::result::Result<(), Error> {
```

Comment:
This `Result` is not part of the output token stream, so there is no need to change anything on this line :)

Reply:
Ah right, that was search and replace. Reverted :)